mros1_reasoner/config/nav_config.yaml

```diff
@@ -1,11 +1,11 @@
 ontology_pkg: mc_mdl_tomasys
-ontology_path: tomasys.owl
+ontology_path: owl/tomasys.owl
 # where is ros_navigation.owl ??
-domain_ontology_pkg: mros1_reasoner
-domain_ontology_path: owl/ros_navigation.owl
+domain_ontology_pkg: mc_mdl_tomasys
+domain_ontology_path: owl/navigation_domain.owl
```

Comment:
@chcorbato is this OK? I think goes with the changes you made to the ontology but I'm not sure